src/com/facebook/buck/apple/umbrella_header.st

```diff
@@ -0,0 +1,6 @@
+#import \<Foundation/Foundation.h>
+
+FOUNDATION_EXPORT double <framework_name>VersionNumber;
+FOUNDATION_EXPORT const unsigned char <framework_name>VersionString[];
+
+<exported_headers:{x| #import \<<x>><\n>}>
```

Comment:
I would like to be able to define module map myself, as we have a lot of Objective-C, Objective-C++, and C++ headers, and to define module correctly for library like that, we have to specify correct `requires objc/cplusplus` option for each header. This will not be possible with auto generated header.